src/Gui/ViewProviderDragger.h

```diff
@@ -113,7 +113,7 @@ class GuiExport ViewProviderDragger : public ViewProviderDocumentObject
      */
     virtual TaskView::TaskDialog* getTransformDialog();
 
-    CoinPtr<SoTransformDragger> transformDragger = nullptr;
+    CoinPtr<SoTransformDragger> transformDragger;
```

Comment:
I don't know the specifics of `CoinPtr<T>`, can you explain why this initialization was removed?

Reply:
This was discussed in the discord server. Basically CoinPtr inherits from intrusive_ptr which ensures that the internal pointer in null initialized in the default constructor. If we don't remove this assignment then we can't forward declare.